kubeadm/KubeCluster.ps1

```diff
@@ -308,7 +358,7 @@ elseif ($Reset.IsPresent)
     UninstallKubelet
     UninstallKubernetesBinaries -Destination  $Global:BaseDir
 
```

Comment:
Can we also clean up the registered task here?
`Unregister-ScheduledTask -TaskName "containerbootstrap"`

If this isn't done then users who had an  erroneous configuration once will not be able to overwrite the stale task upon subsequent installation.

Reply:
Working on check/fix for this.  Should have an additional commit by EOD.